solutions/src/test/java/ru/sberbank/school/task08/JacksonManagerTest.java

```diff
@@ -0,0 +1,50 @@
+package ru.sberbank.school.task08;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JacksonManagerTest extends ManagerTest{
+
+    private static JacksonManager manager;
+    private static MapState<GameObject> mapState;
+
+    @BeforeAll
+    static void init() {
+        manager = new JacksonManager(PATH);
+        manager.initialize();
+
+        List<GameObject> gameObjects = new ArrayList<>();
+        gameObjects.add(manager.createEntity(InstantiatableEntity.Type.BUILDING,
+            InstantiatableEntity.Status.SPAWNED, 100));
+        gameObjects.add(manager.createEntity(InstantiatableEntity.Type.ENEMY,
+            InstantiatableEntity.Status.SPAWNED, 300));
```

Comment:
&#39;InstantiatableEntity&#39; has incorrect indentation level 12, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-379161007>*